### PR TITLE
Create doc about PHP

### LINKED ISSUE #7: Create doc about PHP

### PR DESCRIPTION
**Pull request by swacisko**
Introduction to PHP.
Fixes #7

### CODE REVIEW

**Comment by kvas-damian** [comment]
@swacisko 
For future, to make sure that your PR isn't forgotten or not noticed make sure you assign reviewers or at least ping @bethinkpl/bit

File: php.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+ * explain the process of generating HTML code ‘by’ PHP script
+ * enumerate at least 6 basic data types used in PHP
+ * create conditional statements and loops (loop through indexed and associative arrays)
+ * create your own function
+ * create your own class
+ * name at least 3 superglobal variables
+
+#### Intermediate aka _It means i can really use it_
+
+ * use require() and include() directives to use contents of other PHP files
```
Comment (kvas-damian) [inline php.md]: In higher levels, you can mention about autoloading classes.

File: php.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## Sources:
+
+For those who like documentation-like tutorials
+[Brief and shallow description of php](https://www.w3schools.com/php/)
+
+
+For those who like learning from books:
+[Luke Welling, Laura Thomson, “PHP i MySQL. Tworzenie stron WWW. Vademecum profesjonalisty”](http://miroslawzelent.pl/ksiazki-warte-uwagi/)
```
Comment (kvas-damian) [inline php.md]: I think we should avoid sources in PL language. The idea of BIT is to be used globally so EN is the most reasonable choice. Please link to the EN version of the book.

File: php.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+ * use require() and include() directives to use contents of other PHP files
+ * read/write data from/to a file
+ * connect to external database using PDO
+ * handle possible exceptions where necessary
+
+
+## Sources:
+
+For those who like documentation-like tutorials
+[Brief and shallow description of php](https://www.w3schools.com/php/)
```
Comment (kvas-damian) [inline php.md:33]: http://php.net/manual/en/ is in my opinion also a good source of knowledge for PHP

**Comment by kvas-damian** [comment]
Do you plan to add move levels?

**Comment by swacisko** [comment]
I do not plan to add more levels yet. I may add them in due course (when i reach higher level).

File: my_sql.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# MySQL
```
Comment (rogatty) [inline my_sql.md]: It's not a PR for MySQL, please keep it out of this branch.

File: php.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+## Prerequisite knowledge:   
+HTML
+
+
+## Classification / Acceptance :
+
+#### Apprentice aka _Hello World_
+ * what kind of language PHP is?
+ * explain the process of generating HTML code ‘by’ PHP script
```
Comment (rogatty) [inline php.md]: Quoting https://github.com/bethinkpl/bit-docs/blob/master/!document_template.md:
>Questions students could ask themeselves to evaluate their competence level.

Please reformulate all the points to questions students could ask themselves. You can take a look at other docs for inspiration.

File: php.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+ * I can create my own function
+ * I can create my own class
+ * I can name at least 3 superglobal variables
+
+#### Intermediate aka _It means i can really use it_
+
+ * I can use require() and include() directives to use contents of other PHP files
+ * I can read/write data from/to a file
+ * I can connect to external database using PDO
+ * I can handle possible exceptions where necessary
```
Comment (bognix) [inline php.md:27]: maybe a word about namespaces?

File: php.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+#### Apprentice aka _Hello World_
+ * I know what kind of language PHP is
+ * I can explain the process of generating HTML code ‘by’ PHP script
+ * I can enumerate at least 6 basic data types used in PHP
+ * I can create conditional statements and loops (loop through indexed and associative arrays)
+ * I can create my own function
+ * I can create my own class
+ * I can name at least 3 superglobal variables
+
+#### Intermediate aka _It means i can really use it_
```
Comment (kvas-damian) [inline php.md:19]: I think it's valuable to mention "inheritance" at this level.

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**